tutor/src/screens/assignments/ux.js

```diff
@@ -105,7 +105,8 @@ export default class AssignmentUX {
   }
 
   get formValues() {
-    return this.plan; // .serialize();
+    const templateId = get(this.gradingTemplates, '[0].id');
+    return merge(this.plan, { grading_template_id: templateId });
```

Comment:
@nathanstitt Formik doesn't seem to set `grading_template_id` to what the initially selected option is when it loads, otherwise I wouldn't need this to set this explicitly here for new forms. Ideas/thoughts?

Reply:
ah, I see.  in that case let's set it in the ux.initialize method after we load the templates.

Reply:
Because ux is initialized higher up, form isn't set until the `renderStep`. Should we do something like this: 

```
@action.bound renderStep(form) {
    this.form = form;

    if (this.plan.isNew && this.isShowingDetails) {
      this.form.values.grading_template_id =
        this.form.values.grading_template_id || get(this.gradingTemplates, '[0].id');
    }

    return <Step ux={this} />;
  }
```

Reply:
Sorry realized what you meant now, we could just set it on the plan itself during initialize, which is nice.